docs/pages/guides/devtools.mdx

```diff
@@ -0,0 +1,174 @@
+---
+meta:
+  title: "DevTools"
+  parentTitle: "Guides"
+  description:
+    "Learn how to leverage Liveblocks DevTools to inspect and troubleshoot your
+    collaborative application."
+---
+
+Our [DevTools](/devtools) are a browser-based extension that integrates with
```

Comment:
```suggestion
Our [DevTools](/devtools) is a browser-based extension that integrates with
```

I check and I found some examples where product consider DevTools to be singular and not pluyral. Like [Google](https://support.google.com/adsense/answer/10858959?hl=en) and [React](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en)

> Chrome DevTools **is** a set of web developer tools built directly into the Google Chrome browser.
> React Developer Tools **is** a Chrome DevTools extension for the open-source React JavaScript library